conf/messages

```diff
@@ -1373,6 +1373,20 @@ tai.taxCollected.atSource.otherIncome.description=Tax on other income not collec
 tai.taxCollected.byPAYE.title=Tax collected through your tax code
 tai.taxCollected.byPAYE.description=Pay As You Earn (PAYE) does not include tax paid elsewhere. The tax you’ll pay through PAYE is £{0}.
 
+
+# tax code change
+taxCode.change.journey.preHeading= Your tax code change
+taxCode.change.whatHappensNext.title=What happens next
+taxCode.change.whatHappensNext.paragragh1=If the information about your tax code is right, you do not need to do anything.
+taxCode.change.whatHappensNext.paragragh2=You can check {0} to see how much tax we think you will pay.
+taxCode.change.whatHappensNext.paragragh3=You can {0} in your PAYE Income Tax summary. If this affects your tax code, you will get a new one.
+taxCode.change.whatHappensNext.yourIncomeTaxEstimate.link=your Income Tax estimate
+taxCode.change.whatHappensNext.updateCurrentIncomeOrBenefits.link=update your current income or benefits
+taxCode.change.whatHappensNext.wrongInformation.text=If the information you have seen is wrong
+taxCode.change.yourTaxFreeAmount.title=Your tax-free amount
```

Comment:
Already in messages file

Reply:
Discussed and will leave duplicates for the time being.